dask_cuda/dgx.py

```diff
@@ -1,18 +1,18 @@
 import os
 
-from dask.distributed import Nanny, SpecCluster, Scheduler
-from distributed.system import MEMORY_LIMIT
+from dask.distributed import SpecCluster, Scheduler
 
-from .local_cuda_cluster import cuda_visible_devices
-from .utils import CPUAffinity, get_cpu_affinity
+from .worker_spec import worker_spec
 
 
 def DGX(
-    interface="ib",
+    interface="enp1s0f0",
```

Comment:
I think that Python usually finds the ethernet interface by default.  I wonder if maybe now we can drop this keyword altogether.

Reply:
I think we can make it `None` by default, but some machines may have multiple ethernet interfaces available, so it's generally good to be able to specify that.

Reply:
Addressed in the latest commit.

Reply:
Did we want this to be `None`?